Gopkg.lock

```diff
@@ -349,6 +406,14 @@
   revision = "0ff49de124c6f76f8494e194af75bde0f1a49a29"
   version = "v1.1.6"
 
+[[projects]]
```

Comment:
Not really knowing how dep works, but I'm assuming these are transitive dependencies of things like the test framework. Do they need to be explicitly specified here?

Reply:
this was generated automatically by running `dep ensure`, I wouldnt touch this stuff by hand.